server/tserver/src/test/java/org/apache/accumulo/tserver/log/LogFileKeyTest.java

```diff
@@ -98,4 +103,48 @@ public void testSortOrder() {
     }
 
   }
+
+  private void testToFromKey(int tabletId, long seq) throws IOException {
+    var logFileKey = nk(LogEvents.DEFINE_TABLET, tabletId, seq);
```

Comment:
Would it make sense to loop through all of the LogEvent types here, running the test for each one?

Reply:
I think the different events and sorting get covered enough in `SortedLogRecoveryTest`.